ClientApp/src/app/constants/sentence-type.constants.ts

```diff
@@ -0,0 +1,5 @@
+export const sentenceTypeConstants = {
+    english: 'English',
```

Comment:
```suggestion
    ENGLISH: 'English',
```
By convention these should be all caps

Reply:
Fixed!